ui-v2/app/services/repository/node.js

```diff
@@ -1,9 +1,17 @@
 import RepositoryService from 'consul-ui/services/repository';
 import { inject as service } from '@ember/service';
+import { get } from '@ember/object';
+
 const modelName = 'node';
 export default RepositoryService.extend({
   coordinates: service('repository/coordinate'),
   getModelName: function() {
     return modelName;
   },
+  findByLeader: function(dc) {
+    const query = {
+      dc: dc,
```

Comment:
```suggestion
      dc,
```

Reply:
Hey @gregone ,

Thanks for looking at this! I was thinking of merging without doing this suggestion, reasons being:

Most of the Consul UI codebase is ES5-y (think ember 2.18 timescales here, 2-3 years back), we generally don't do this kind of assignment (I think the only ES6-y thing we use is the Object.assign-like `{...{}, ...{}}` which was only because we preferred that to having to use the ember provided `assign`)

Other examples specifically in Repositories are:

https://github.com/hashicorp/consul/blob/d84863799deca45ccf4bec5ab9f645ccae6b3aeb/ui-v2/app/services/repository/token.js#L27-L30

https://github.com/hashicorp/consul/blob/d84863799deca45ccf4bec5ab9f645ccae6b3aeb/ui-v2/app/services/repository/session.js#L12-L15

https://github.com/hashicorp/consul/blob/d84863799deca45ccf4bec5ab9f645ccae6b3aeb/ui-v2/app/services/repository/proxy.js#L13-L16

https://github.com/hashicorp/consul/blob/d84863799deca45ccf4bec5ab9f645ccae6b3aeb/ui-v2/app/services/repository/kv.js#L27-L29

..but there are a few more around and about.

I can completely see where you are coming from, right now it's one of those consistency/context things. Once we get past ember 3.1* we'll think about moving up a notch, and if we do we'll probably skim the entire codebase in one go everything is consistent.

Anyway, let me know what you think, happy to add this before merging if you feel strongly about it.

Thanks!

Reply:
Hey @gregone 

We are going to try and get this into our next release, so I'm going to merge this without the suggestion, hope that's ok, thanks for reviewing!